API/DataModel/DataContext.cs

```diff
@@ -25,6 +30,23 @@ protected override void OnModelCreating(ModelBuilder builder)
 
             builder.Entity<Role>().HasMany(r => r.UserRoles).WithOne(r => r.Role)
                                   .HasForeignKey(r => r.RoleId).IsRequired();
+
+            //clustered keys
+            builder.Entity<OrderToServicePrice>()
+                   .HasKey(c => new { c.OrderId, c.ServicePriceId});
+
+            //seed dictionaries
+
+            builder.Entity<OrderStatus>().HasData(
```

Comment:
zajebiste, nie znałem tego. Przerobie potem seedowanie ról w ten sam sposób.

Reply:
Jeżeli jest więcej danych to lepiej pobrać z JSONa lub csva twoim sposobem. Też tak seeduję w pracy. W tym wypadku jest tylko 5 linijek i będzie to wykonywane tylko raz więc nie ma co się spinać.